tcf_website/static/landing/feedbackform.js

```diff
@@ -1,4 +1,8 @@
 function submit(event) {
+    postToDiscord(event);
```

Comment:
Is it necessary to create a new function and then call it if that's the only thing that happens in `submit`? Maybe it's a readability thing, but I feel like that both isn't that big of a problem and could be addressed via comment. Not a big deal though.

Reply:
will add in validation here tho so it'll be more than just that soon :)

Reply:
I guess the idea is that submit could have more functionality down the line

Reply:
yup, my stuff will be in a sep PR tho comin soon